### ISSUE TITLE
doesn't work on windows 10

### ISSUE THREAD

**Issue by pedrorochaorg** [psychok7/django-yadpt-starter#31]
I've created a folder named "backend" and i've cloned the github repo into it.
Inside the folder "backend" are the same files that are present in the master branch of this repo.

if i run the start cmd inside the "backend" folder i get the following output:

![khantemplate2](https://cloud.githubusercontent.com/assets/10233181/23519297/51e50a5e-ff6e-11e6-81bf-eac8e814288d.PNG)

if i run the start cmd in the "backend" parent folder i get the following output

![khantemplate1](https://cloud.githubusercontent.com/assets/10233181/23519286/4668be50-ff6e-11e6-87c6-343f30f5e83e.PNG)

**Comment by psychok7** [comment]
@pedrorochaorg I will have look as soon as possible. Do you have a Linux VM to test it and see if it works there?

**Comment by pslobo** [comment]
Yeah, I'm thinking this has to do with filesystem issues. We didn't really test on windows systems.

**Comment by psychok7** [comment]
@pedrorochaorg indeed, i think the first step for us to eliminate the possibility of "human error" is for you to try it inside a linux VM. if it works, then we know we must fix the windows version.

This is also a workaround if this is in fact a windows bug. after creating the project inside the linux vm, you can copy paste it to windows and keep on working (everything else should work).

After that, later on you, me and @pslobo can figure out whats breaking and the best way to fix it

**Comment by pedrorochaorg** [comment]
@pslobo @psychok7 i can confirm you that it works inside a Linux VM

**Comment by psychok7** [comment]
@pedrorochaorg good, the copy paste workaround should allow you to continue working meanwhile, until we find some time to fix it on windows.

@pslobo i will take care of windows, can you test on MacOs ?

**Comment by psychok7** [comment]
@pslobo https://github.com/psychok7/django-yadpt-starter/issues/32

**Comment by psychok7** [comment]
Honestly, i don't see myself fixing this bug until we release a major version (that implies an actual change on the starter script) or if we get more Windows users complaints (sorry guys). If we had proper unit tests i could "trust" making a change without manually testing everything.

This issue will stay open for future reference and the presented workaround will be the OFFICIAL workaround until someone (probably me) addresses this issue.
